GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.Designer.cs

```diff
@@ -50,19 +51,22 @@ private void InitializeComponent()
             this.tableLayoutPanelForDiffViewer.ColumnCount = 2;
             this.tableLayoutPanelForDiffViewer.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
             this.tableLayoutPanelForDiffViewer.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
-            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkShowDiffForAllParents, 0, 6);
-            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkOpenSubmoduleDiffInSeparateWindow, 0, 5);
             this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkRememberIgnoreWhiteSpacePreference, 0, 0);
             this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkRememberShowNonPrintingCharsPreference, 0, 1);
             this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkRememberShowEntireFilePreference, 0, 2);
             this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkRememberNumberOfContextLines, 0, 3);
-            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkOmitUninterestingDiff, 0, 4);
-            this.tableLayoutPanelForDiffViewer.Controls.Add(this.label1, 0, 7);
-            this.tableLayoutPanelForDiffViewer.Controls.Add(this.VerticalRulerPosition, 1, 7);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkRememberShowSyntaxHighlightingInDiff, 0, 4);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkOmitUninterestingDiff, 0, 5);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkOpenSubmoduleDiffInSeparateWindow, 0, 6);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkShowDiffForAllParents, 0, 7);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.label1, 0, 9);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.VerticalRulerPosition, 1, 9);
             this.tableLayoutPanelForDiffViewer.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.tableLayoutPanelForDiffViewer.Location = new System.Drawing.Point(0, 0);
+            this.tableLayoutPanelForDiffViewer.Location = new System.Drawing.Point(0, 0);   
```

Comment:
Nitpick: Avoid whitespace at end of line, please.
```suggestion
            this.tableLayoutPanelForDiffViewer.Location = new System.Drawing.Point(0, 0);
```